snmp/tests/test_profiles.py

```diff
@@ -520,8 +520,14 @@ def test_cisco_3850(aggregator):
             aggregator.assert_metric('snmp.{}'.format(metric), metric_type=aggregator.GAUGE, tags=tags, count=1)
         aggregator.assert_metric('snmp.cieIfResetCount', metric_type=aggregator.MONOTONIC_COUNT, tags=tags, count=1)
 
-    for source in range(1, 3):
-        env_tags = ['power_source:{}'.format(source)]
+    power_supplies = [
+        (1, 'Switch 1 - Power Supply B, NotExist'),
+        (1, 'Switch 2 - Power Supply B, NotExist'),
+        (2, 'Switch 1 - Power Supply A, Normal'),
+        (2, 'Switch 2 - Power Supply A, Normal'),
```

Comment:
Note that the Agent and backend will strip/standardize/lowercase some characters.